Assets/HoloToolkit/XAML/Scripts/ApplicationViewManager.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UnityEngine;
+#if WINDOWS_UWP
```

Comment:
I don't believe that this is a valid platform definition.  See https://docs.unity3d.com/Manual/PlatformDependentCompilation.html

Use: `# UNITY_WSA`